### ISSUE TITLE
Bundler 2.2.15 doesn't use prepackaged libv8 version; tries to build locally

### ISSUE THREAD

**Issue by themilkman** [rubygems/rubygems#4513]
### Describe the problem as clearly as you can

Hello, I have massive problems with bundler > 2.1.

On my Kubuntu 20.10 (amd64 notebook) machine as well while our capistrano deployment on an Debian GNU/Linux 10 (buster) amd64 virtual server using Ruby 3.0.1 and bundler 2.2.15 the bundle process is fetching all the libv8 sources (1-2GBytes btw) and tries to compile them. This fails (for reasons I better do not want to understand at this point), but the core issue is that it doesn't use the prebuild packages (e.g.  8.4.255.0 x86_64-linux, see https://rubygems.org/gems/libv8/versions ).

When running `bundle _2.1.4_ update` locally I am able to update/install my dependencies since it uses the prebuild gem version.

I think this is related to the issues mentioned here: https://github.com/rubyjs/libv8/issues/310#issuecomment-742481001

Since the old bundler version seems to work fine I'd say that the behavior isn't intended, is it?

Thanks for any hints!

### Post steps to reproduce the problem

Run bundle update on a project using e.g. libv8 with Ruby 3.0.1 (might be optional) an bundler 2.2.15

### What were you expecting to happen?

Bundler successfully installs libv8 using the prebuilt version

### What actually happened?

Downloads the sources and tries to compile them.

## Environment

```
Bundler             2.2.15
  Platforms         ruby, x86_64-linux
Ruby                3.0.1p64 (2021-04-05 revision 0fb782ee38ea37fd5fe8b1f775f8ad866a82a3f0) [x86_64-linux]
  Full Path         /home/myuser/.rvm/rubies/ruby-3.0.1/bin/ruby
  Config Dir        /home/myuser/.rvm/rubies/ruby-3.0.1/etc
RubyGems            3.2.15
  Gem Home          /home/myuser/.rvm/gems/ruby-3.0.1
  Gem Path          /home/myuser/.rvm/gems/ruby-3.0.1:/home/myuser/.rvm/rubies/ruby-3.0.1/lib/ruby/gems/3.0.0
  User Home         /home/myuser
  User Path         /home/myuser/.gem/ruby/3.0.0
  Bin Dir           /home/myuser/.rvm/gems/ruby-3.0.1/bin
Tools               
  Git               2.31.1
  RVM               1.29.12 (latest)
  rbenv             not installed
  chruby            not installed
  rubygems-bundler  (1.4.5)
```

## Bundler Build Metadata

```
Built At          2021-03-19
Git SHA           3dbcc68293
Released Version  true
```

## Bundler settings

```
jobs
  Set for the current user (/home/myuser/.bundle/config): 3
gem.test
  Set for the current user (/home/myuser/.bundle/config): false
gem.mit
  Set for the current user (/home/myuser/.bundle/config): false
gem.coc
  Set for the current user (/home/myuser/.bundle/config): false
default
  Set for the current user (/home/myuser/.bundle/config): "2.1.4"
```

**Comment by deivid-rodriguez** [comment]
Hi! Can you try bundler 2.2.14? I fixed a regression in 2.2.15 that I haven't released yet, I plan to do that in the next few days.

**Comment by themilkman** [comment]
Hi, thanks for your amazing fast response! :-)

Locally it seems to work with `bundle _2.2.14_ update` (using 3.0.0+3.0.0+2.7.3).

`Installing libv8 8.4.255.0 (x86_64-linux)` and also 
`Installing nokogiri 1.11.2 (x86_64-linux)` worked as expected.

I couldn't test it on the debian server since I failed to change the default-bundler version and had no time to investigate other ways to inject this into the capistrano deployment, yet.

So my guess would be: yes, seems to be 2.2.15 only.

Can we hope for a soon release of 2.2.16 where the issue is hopefully gone?

Thanks again!

**Comment by deivid-rodriguez** [comment]
Yes, this was fixed by https://github.com/rubygems/rubygems/pull/4497, and I expect to release it in a couple of days.

**Comment by deivid-rodriguez** [comment]
Closing since this is fixed.

**Comment by themilkman** [comment]
Seems to be fixed with 2.2.16, thank you very much!

**Comment by deivid-rodriguez** [comment]
No problem!
